src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const message = document.createElement('div');
+const body = document.body;
+
+new Promise((resolve, reject) => {
```

Comment:
you can just omit `reject`
```suggestion
new Promise((resolve) => {
```